code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -181,6 +181,11 @@
 	results = list("golemmutationtoxin" = 1)
 	required_reagents = list("unstablemutationtoxin" = 1, "silver" = 1) //not too hard to get but also not just there in xenobio
 
+/datum/chemical_reaction/vox_mutation_toxin
```

Comment:
Personally, I think we should take a look at these things and kick quite a few of them out. Turning someone into a Plasmaperson or a Vox out of sudden will kill them really quickly, and I don't think we should hand people an easy way to do that, especially when going back on it is so hard.

Reply:
I think if someone spends enough time to get the slimes and make these sorts of things, then it's a decent tool as a traitor. They're using it on others as a non-antag, then that's some pretty clear self-antagging and worthy of punishment.